src/framework/standard/mod.rs

```diff
@@ -1142,3 +1143,17 @@ impl fmt::Display for HelpBehaviour {
        }
     }
 }
+
+impl HelpBehaviour {
+    /// Merges two behaviours so that the more effectful behavior is preserved.
+    pub fn merge(self, hb: HelpBehaviour) -> HelpBehaviour {
```

Comment:
Ok. The code I had looked at seemed to be leaning towards shorter names, but I'm happy to be clearer. :)

I was considering just making this an impl for `std::ops::Add` or `std::ops::BitAnd`, but wanted to get your thoughts.

Reply:
Totally understandable, I'm personally a fan of descriptive names, other bits originate from other people, I guess : )

Yes, as long as you describe the code-behaviour as much as needed, I think it would be interesting to add `std::ops::BitAnd` as `std::ops::Add` might cause confusion.